spec/support/override_facts.rb

```diff
@@ -0,0 +1,30 @@
+# Override facts
```

Comment:
I intend to submit this helper to voxpupuli-test since it's likely to be merged quickly there. Just submitting it here to see if all tests pass with it.

Reply:
Blocked on https://github.com/voxpupuli/voxpupuli-test/pull/16